docs/interacting/web-ui.md

```diff
@@ -0,0 +1,35 @@
+# Web application
+
+The web application is hosted at [skywalking-banyandb-webapp](http://localhost:17913/) when you boot up the BanyanDB server.
+The web ui could use to query data stored in streams, measures, and properties.
+
+## Stream
+![web-ui-stream.png](https://skywalking.apache.org/doc-graph/banyandb/v0.7.0/web-ui/web-ui-stream.png)
+
+The stream page shows the data stored in the stream. The menu on the left side shows the list of streams, index-rule, index-rule-binding and organized by their groups. 
+
+The user can select the stream to view the data, the data is displayed in a table format and can query/filter the data by the following:
+- Select the `Time Range` to be queried. 
+- Select the `Tag Families` to be queried.
+- Modify/Write the query config directly on the page. The config can refer to the [bydbctl query stream](./bydbctl/query/stream.md).
+
+## Measure
+![web-ui-measure.png](https://skywalking.apache.org/doc-graph/banyandb/v0.7.0/web-ui/web-ui-measure.png)
```

Comment:
Like the `Stream,` schema-relevant operations are supported on this page.

Reply:
More than the index-rule and index-rule-binding, topn-aggregation could be managed in this page.